### DISCUSSION TITLE
Which panel on grafana shows QPS of searching request?

### DISCUSSION THREAD

**Discussion by xiaojuan8** [milvus-io/milvus#22025] [Q&A and General discussion] [answered]
hi, I want to test my milvus cluster with performance obervation. And I want to know which panel on grafana shows QPS for search request?

I just found this one, but I am not sure :
![image](https://user-images.githubusercontent.com/74544828/217169034-f7cd9f8f-c05b-44c2-86bb-a859ca78c8b4.png)

**Comment by binbinlv** [comment, marked as answer]
Here I think what you want is Search Request Rate in Query Node:  The total number of search and query requests received per second
![image](https://user-images.githubusercontent.com/83755740/217175292-da9e48b6-c2cc-4a84-819a-46201de1cbe6.png)

Search Vector Count Rate is VPS： The average number of vectors queried per second 

And by the way, https://github.com/milvus-io/milvus/tree/master/deployments/monitor/grafana here is the detail explanation for each metric which you could have a reference.

**Reply by xiaojuan8** [reply]
Thanks, I get it.
